test/30_debate/adminstatisticfunctions.js

```diff
@@ -0,0 +1,78 @@
+import {SocketConfig} from '../../src/conf/config.js'
+import {DebateManager} from "../../src/debatemanager.js";
+import io from 'socket.io-client'
+import chai from 'chai';
+
+const expect = chai.expect;
+const should = chai.should();
+
+const SERVER_ADDRESS = `http://localhost:${SocketConfig.SOCKET_PORT}`;
+const PRIVILEGED_NAMESPACE = `${SERVER_ADDRESS}${SocketConfig.PRIVILEGED_NAMESPACE}`;
+
+describe('Statistic functions test', () => {
+    let debateManager;
+    let admin;
+
+    describe("Get stats", () => {
+        before(async ()  => {
+            debateManager = new DebateManager();
+            await debateManager.start();
+
+            admin = io.connect(`${PRIVILEGED_NAMESPACE}`, {
+                path: SocketConfig.DEFAULT_PATH,
+                forceNew: true,
+                query: {
+                    password: `${SocketConfig.ADMIN_PASSWORD}`,
```

Comment:
CWE 798 ( _L O L_ )

Reply:
Corrected it is now a password of 256 non-ASCII characters and I'm the only one that knows it